<?xml version="1.0"?>
<launch>

    <arg name="camera" default="/iiwa/camera1"/>
    <include file="$(find aruco_ros)/launch/usb_cam_aruco.launch">
        <arg name="camera" value="$(arg camera)" />
    </include>

    <rosparam command="load" file="$(find kdl_ros_control)/config/param.yaml"/>

    <arg name="urdf" default="$(find iiwa_description)/urdf/iiwa14.urdf"/> 
    <param name="urdf" value="$(arg urdf)"/>
    <node name="kdl_ros_control_node" pkg="kdl_ros_control" type="kdl_robot_test" output="screen"/>

    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/aruco_single/result"/>
        <param name="autosize" value="true" />
    </node>

</launch>